data/json/items/gun/40mm.json

```
@@ -180,5 +180,28 @@
     "clip_size": 3,
     "modes": [ [ "DEFAULT", "single", 1, "NPC_AVOID" ], [ "MULTI", "multi", 3, [ "NPC_AVOID", "SIMULTANEOUS" ] ] ],
     "proportional": { "weight": 1.5, "volume": 1.8, "price": 2 }
+  },
+  {
+    "id": "pseudo_m203",
+    "copy-from": "launcher_base",
+    "looks_like": "m203",
+    "type": "GUN",
+    "name": "M203 array",
+    "description": "Six M203 grenade launchers for use on an automated 40mm turret.",
+    "weight": "8200 g",
+    "volume": "4 L",
+    "price": 250000,
+    "to_hit": -1,
+    "bashing": 14,
+    "material": "steel",
+    "ammo": "40mm",
+    "range": 30,
+    "ranged_damage": 1,
+    "dispersion": 300,
+    "durability": 10,
+    "reload": 400,
+    "modes": [ [ "DEFAULT", "semi-auto", 1, "NPC_AVOID" ] ],
+    "magazines": [ [ "40mm", [ "belt40mm" ] ] ],
```

```suggestion
    "clip_size": 16,
```

While I realize a manual array of m203's is a bit anticlimactic compared to a MK19, I don't think it should be using mk19 belts; I think you should just bump up the number of launchers to whatever capacity you need.